libc/src/__support/OSUtil/linux/aarch64/vdso.h

```diff
@@ -0,0 +1,36 @@
+//===---------- aarch64 vdso configuration ------------------------* C++ *-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#ifndef LLVM_LIBC_SRC___SUPPORT_OSUTIL_LINUX_AARCH64_VDSO_H
+#define LLVM_LIBC_SRC___SUPPORT_OSUTIL_LINUX_AARCH64_VDSO_H
+#include "src/__support/CPP/string_view.h"
```

Comment:
you should also include the top level vdso header (`src/__support/OSUtil/linux/vdso.h`). Each header should be able to build on its own.

Reply:
That would make a recursive include chain, which is not good. (it wouldn't actually infinitely recurse because of header guards, but it's still a bad idea).

Perhaps instead these arch/vdso.h files should be renamed to ".inc" and be without header guards, to show that they're not intended to be standalone headers?

Reply:
Or we can refactor `VDSOSym` to its own header file `vdsosym.h` to be included in all targets and the main `vdso.h`.